python/paddle/v2/fluid/optimizer.py

```diff
@@ -48,26 +56,33 @@ def __init__(self, learning_rate, global_step=None, regularization=None):
         self.helper = None
 
     def _create_global_learning_rate(self):
-        if isinstance(self._global_learning_rate, float):
-            self._global_learning_rate = layers.create_global_var(
-                name=unique_name.generate("learning_rate"),
-                shape=[1],
-                value=float(self._global_learning_rate),
-                dtype='float32',
-                persistable=True)
-
-        if not isinstance(self._global_learning_rate, framework.Variable):
-            raise ValueError("learning rate should be a Variable, "
-                             "actual type is %s",
-                             type(self._global_learning_rate))
-
-    @property
-    def global_learning_rate(self):
+        lr = self.global_learning_rate()
+
+        if isinstance(lr, framework.Variable):
+            return
+        else:
```

Comment:
`else` is not needed, since it has been returned in if block.

Reply:
`else` is used to do type check and give some useful information for code reader